contracts/ReputationMiningCycleCommon.sol

```diff
@@ -0,0 +1,105 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.5.8;
+pragma experimental "ABIEncoderV2";
+
+import "../lib/dappsys/math.sol";
+import "./PatriciaTree/PatriciaTreeProofs.sol";
+import "./ReputationMiningCycleStorage.sol";
+import "./ITokenLocking.sol";
+import "./IColonyNetwork.sol";
+
+
+contract ReputationMiningCycleCommon is ReputationMiningCycleStorage, PatriciaTreeProofs, DSMath {
+  /// @notice Minimum reputation mining stake in CLNY
+  uint256 constant MIN_STAKE = 2000 * WAD;
+
+  function rewardResponder(address _responder) internal returns (bytes32) {
+    respondedToChallenge[_responder] = true;
+    uint256 reward = disputeRewardIncrement();
+    ITokenLocking(tokenLockingAddress).reward(
+      _responder,
+      reward
+    );
+    rewardsPaidOut += reward;
+  }
+
+  function disputeRewardIncrement() internal view returns (uint256) {
```

Comment:
This name to me suggests state mutation (i.e. "increment the dispute reward"). Can we specify `calculateDisputeRewardIncrement`?

Reply:
`disputeRewardSize`?

Reply:
I like a verb prefix like `calculate...` or `get...` since it implies that we're fetching some value for use elsewhere. `disputeRewardSize` is fine though.